development/metamaskbot-build-announce.js

```diff
@@ -15,6 +15,21 @@ function capitalizeFirstLetter(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+function getHumanReadableSize(bytes) {
+  if (!bytes) {
+    return '0 Bytes';
+  }
+
+  const k = 1024;
+  const sizes = ['Bytes', 'KiB', 'MiB'];
+  const i = Math.floor(Math.log(Math.abs(bytes)) / Math.log(k));
+  return `${parseFloat((bytes / Math.pow(k, i)).toFixed(2))} ${sizes[i]}`;
```

Comment:
As a math function:
$`x=\frac{y}{\left(1024^{\log\left(\frac{\left(\left|y\right|\right)}{\log\left(1024\right)}\right)}\right)}`$

Reply:
This might be the first time I've ever seen anyone use a rendered math equation in a GitHub comment 😂